app/VLC.Universal/VLC.Universal.csproj

```diff
@@ -1647,6 +1647,7 @@
     <Warning Text="$(MSBuildProjectFile) is Multilingual build enabled, but the Multilingual App Toolkit is unavailable during the build. If building with Visual Studio, please check to ensure that toolkit is properly installed." />
   </Target>
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it.
```

Comment:
This extra line you probably did not want to commit breaks the csproj (and project loading in VS).